src/main/java/seedu/duke/util/Parser.java

```diff
@@ -172,7 +179,7 @@ public Command parseCommand(String userInput) {
      * Checks if an input string if blank. If it is, throw the provided system exception. If it is not, return that
      * string trimmed.
      *
-     * @param input Input to be checked.
+     * @param input         Input to be checked.
```

Comment:
As mentioned previously, I don't think the other JavaDoc comments had white space. Maybe we should standardize?

Reply:
Im not the author for this function. Ill make the changes.

Reply:
I know why already. the ide by default turns on whose whitespaces when auto format